### DISCUSSION TITLE
They both switch modeN to modeI, one is highlighted and one is not?

### DISCUSSION THREAD

**Discussion by 6lj6** [helix-editor/helix#6674] [Q&A]
`i` is without highlighting on insertion while `a` is with highlighting on added content.

![image](https://user-images.githubusercontent.com/44439620/230779067-61799aac-2ea9-4b20-b222-bbbef49ac8bf.png)

![image](https://user-images.githubusercontent.com/44439620/230779146-1ce228e4-c245-4fae-9c97-f647c4a9f965.png)

How to config it?

What's the advantages of this design?

**Comment by the-mikedavis** [comment]
`a` extends the selection to include the new characters that you insert. This is by design and is useful when working with multiple selections, see https://github.com/helix-editor/helix/discussions/2924
